ios/Podfile

```diff
@@ -65,7 +65,7 @@ post_install do |installer|
           'PERMISSION_SPEECH_RECOGNIZER=0',
 
           ## dart: PermissionGroup.photos
-          # 'PERMISSION_PHOTOS=0',
+          'PERMISSION_PHOTOS=1',
```

Comment:
do we need to add more information in `Info.plist`?

Reply:
It was declared before, but we will update it correctly to support videos as well.

<img width="934" alt="Screenshot 2024-09-16 at 09 39 09" src="https://github.com/user-attachments/assets/e25588a5-ef3c-4c67-9c61-771f24aec16d">